src/main/webapp/view/discussion/discussion.js

```diff
@@ -230,14 +261,29 @@ class DiscussionComment extends HTMLElement {
     span.slot = name;
     this.appendChild(span);
   }
+
+  /**
+   * Scroll `ELEMENT_DISCUSSION` such that this element is at the top.
```

Comment:
why quote it instead of using @code?

Reply:
Also you are talking about the code and someone needs to look up what ELEMENT_DISCUSSION is. Why not just say: Scroll the discussion element to the first discussion created. Or something like that.

Reply:
ditto re: @code vs ticks

Reply:
What's ```ELEMENT_DISCUSSION``` sorry? is this the current discussion element in the ui? If so perhaps what we're looking for is 
"Scroll such that this element is at the top"?

Reply:
"Scroll such that this element is at the top of the discussion area."